scripts/check_ecr_findings.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# Exit on any error
+set -eu -o pipefail
+
+# Set the repository name and image tag
+repository_name="$1"
+image_tag="$2"
+
+# Wait for the scan to complete
+echo "Waiting for scan to complete..."
+while true; do
```

Comment:
Can you add some fixed number of retries before we exit with a failure code? Perhaps something like 30 retries? (which would mean a max of 300 seconds, since we `sleep 10` on retry, for a 5m total timeout)?